[ERROR] - 2025-02-16 14:45:55,048 - root - ea020bfd-a29b-41c1-aec0-8158dd5d22a1 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 14:48:42,462 - root - ea020bfd-a29b-41c1-aec0-8158dd5d22a1 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 14:50:07,020 - root - d5cb6f7f-a83e-4d44-9377-aefedf88af4f 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 14:52:34,881 - root - 1923d416-2cac-4d75-a880-57d851d16d62 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 14:52:54,203 - root - 1923d416-2cac-4d75-a880-57d851d16d62 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 14:54:02,585 - root - 1923d416-2cac-4d75-a880-57d851d16d62 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 14:58:02,897 - root - 42a1380e-33ee-473e-a2d4-ae27024cca66 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 15:00:06,418 - root - 42a1380e-33ee-473e-a2d4-ae27024cca66 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 15:00:46,553 - root - f8e8a6aa-20dc-4ecd-ab51-5600f915ee86 'AuthorizationManager' object has no attribute 'is_deleted'
[ERROR] - 2025-02-16 16:29:04,254 - root - 2483a458-235a-44b6-babd-8163ba3b4f40 Could not initialize target column for ForeignKey 'tunnel.id' on table 'interface': table 'tunnel' has no column named 'id'
[ERROR] - 2025-02-16 16:33:22,202 - root - 90b57f25-d922-4dc4-ac02-63a1433e3be9 reverse_property 'device' on relationship Tunnel.interfaces references relationship Interface.device, which does not reference mapper Mapper[Tunnel(tunnel)]
[ERROR] - 2025-02-16 16:35:37,165 - root - e0c26d45-f48f-46a8-af57-1b54d5a41549 (sqlalchemy.dialects.postgresql.asyncpg.Error) <class 'asyncpg.exceptions.DataError'>: invalid input for query argument $7: UUID('d30ea2dc-f805-4693-9822-eeab908512... (expected str, got UUID)
[SQL: INSERT INTO "user" (id, name, password, admin, is_deleted, created_at, created_by, modified_at, modified_by, deleted_at, deleted_by) VALUES ($1::UUID, $2::VARCHAR, $3::VARCHAR, $4::BOOLEAN, $5::BOOLEAN, $6::TIMESTAMP WITHOUT TIME ZONE, $7::VARCHAR, $8::TIMESTAMP WITHOUT TIME ZONE, $9::VARCHAR, $10::TIMESTAMP WITHOUT TIME ZONE, $11::VARCHAR)]
[parameters: (UUID('6b2b2d14-cbbf-4118-83bd-209d383ff5c4'), 'hanoch2', '$2b$12$OGrvJRkMHpsvmGa7SygGI.4XGolfmpX6Es5f1X9Jy6AMm9pSh0tHW', True, False, datetime.datetime(2025, 2, 16, 16, 35, 37, 154560), UUID('d30ea2dc-f805-4693-9822-eeab9085127b'), datetime.datetime(2025, 2, 16, 16, 35, 37, 154560), UUID('d30ea2dc-f805-4693-9822-eeab9085127b'), None, None)]
(Background on this error at: https://sqlalche.me/e/20/dbapi)
[ERROR] - 2025-02-16 16:38:37,596 - root - dcff1bf7-0d1c-4087-a2b6-7c77e4731ceb (sqlalchemy.dialects.postgresql.asyncpg.Error) <class 'asyncpg.exceptions.DataError'>: invalid input for query argument $7: UUID('0f365702-81d3-4b10-a31e-acc6518526... (expected str, got UUID)
[SQL: INSERT INTO "user" (id, name, password, admin, is_deleted, created_at, created_by, modified_at, modified_by, deleted_at, deleted_by) VALUES ($1::UUID, $2::VARCHAR, $3::VARCHAR, $4::BOOLEAN, $5::BOOLEAN, $6::TIMESTAMP WITHOUT TIME ZONE, $7::VARCHAR, $8::TIMESTAMP WITHOUT TIME ZONE, $9::VARCHAR, $10::TIMESTAMP WITHOUT TIME ZONE, $11::VARCHAR)]
[parameters: (UUID('fcda7ff0-9bf6-40d9-9877-525a342afdba'), 'hanoch2', '$2b$12$SXBm2J1gfrGBsqLGPyi6zOV3Buo/xKp3ASQYc.WQyNuurPsMbAqMC', True, False, datetime.datetime(2025, 2, 16, 16, 38, 37, 592330), UUID('0f365702-81d3-4b10-a31e-acc65185260b'), datetime.datetime(2025, 2, 16, 16, 38, 37, 592330), UUID('0f365702-81d3-4b10-a31e-acc65185260b'), None, None)]
(Background on this error at: https://sqlalche.me/e/20/dbapi)
[ERROR] - 2025-02-16 16:39:54,790 - root - a880e695-6c77-47f8-aa96-356e166f0729 (sqlalchemy.dialects.postgresql.asyncpg.Error) <class 'asyncpg.exceptions.DataError'>: invalid input for query argument $7: UUID('f1ffa77e-34f0-4db2-bf27-0ff2c2f65f... (expected str, got UUID)
[SQL: INSERT INTO "user" (id, name, password, admin, is_deleted, created_at, created_by, modified_at, modified_by, deleted_at, deleted_by) VALUES ($1::UUID, $2::VARCHAR, $3::VARCHAR, $4::BOOLEAN, $5::BOOLEAN, $6::TIMESTAMP WITHOUT TIME ZONE, $7::VARCHAR, $8::TIMESTAMP WITHOUT TIME ZONE, $9::VARCHAR, $10::TIMESTAMP WITHOUT TIME ZONE, $11::VARCHAR)]
[parameters: (UUID('0e420bc9-2e50-48a6-8faa-0dbcb841fd9b'), 'hanoch2', '$2b$12$J3WWaAKkD4rgTZIS2xqT7eHHtGfxD/jnFgomsMrp3aOMM13TVuu2e', True, False, datetime.datetime(2025, 2, 16, 16, 39, 54, 786389), UUID('f1ffa77e-34f0-4db2-bf27-0ff2c2f65f61'), datetime.datetime(2025, 2, 16, 16, 39, 54, 786389), UUID('f1ffa77e-34f0-4db2-bf27-0ff2c2f65f61'), None, None)]
(Background on this error at: https://sqlalche.me/e/20/dbapi)
